lib/service_manager.dart

```diff
@@ -102,6 +138,17 @@ class ServiceConnectionManager {
     _stateController.add(null);
     _connectionClosedController.add(null);
   }
+
+  // TODO(kenzie): add hot restart method, register method in flutter_tools.
+
+  Future<void> performHotReload() async {
+    try {
+      await callMulticastService('reloadSources',
+          isolateId: _isolateManager.selectedIsolate.id);
+    } catch (e) {
+      print('Error during hot reload: "$e."');
```

Comment:
Let's add a TODO here to improve error handling (we need a better way for the app in general, and then we can call it from places in the code like this one).

Reply:
done.